eodag/utils/logging.py

```diff
@@ -55,72 +58,41 @@ def setup_logging(verbose: int, no_progress_bar: bool = False) -> None:
-                "disable_existing_loggers": False,
-                "formatters": {
-                    "standard": {
-                        "format": "%(asctime)-15s %(name)-32s [%(levelname)-8s] %(message)s"
-                    }
-                },
-                "handlers": {
-                    "console": {
-                        "level": "DEBUG",
-                        "class": "logging.StreamHandler",
-                        "formatter": "standard",
-                    }
-                },
-                "loggers": {
-                    "eodag": {
-                        "handlers": ["console"],
-                        "propagate": True,
-                        "level": "INFO",
-                    },
-                    "sentinelsat": {
-                        "handlers": ["console"],
-                        "propagate": True,
-                        "level": "INFO",
-                    },
-                },
-            }
-        )
-    elif verbose == 3:
-        logging.config.dictConfig(
-            {
-                "version": 1,
-                "disable_existing_loggers": False,
-                "formatters": {
-                    "verbose": {
-                        "format": (
-                            "%(asctime)-15s %(name)-32s [%(levelname)-8s] (tid=%(thread)d) %(message)s"
-                        )
-                    }
-                },
-                "handlers": {
-                    "console": {
-                        "level": "DEBUG",
-                        "class": "logging.StreamHandler",
-                        "formatter": "verbose",
-                    }
+        return None
+
+    formatter = "standard" if verbose == 2 else "verbose"
+    level = "INFO" if verbose == 2 else "DEBUG"
+
+    logging.config.dictConfig(
+        {
+            "version": 1,
+            "disable_existing_loggers": False,
+            "formatters": {
+                f"{formatter}": {
+                    "format": "%(asctime)-15s %(name)-32s [%(levelname)-8s] %(message)s"
+                }
+            },
+            "handlers": {
+                "console": {
+                    "level": f"{level}",
+                    "class": "logging.StreamHandler",
+                    "formatter": f"{formatter}",
+                }
+            },
+            "loggers": {
+                "eodag": {
+                    "handlers": ["console"],
+                    "propagate": True,
+                    "level": f"{level}",
                 },
-                "loggers": {
-                    "eodag": {
-                        "handlers": ["console"],
-                        "propagate": True,
-                        "level": "DEBUG",
-                    },
-                    "sentinelsat": {
-                        "handlers": ["console"],
-                        "propagate": True,
-                        "level": "DEBUG",
-                    },
+                "sentinelsat": {
```

Comment:
Remove deprecated `sentinelsat` logger and add missing `eodag-cube` handler
```suggestion
                "eodag-cube": {
```

Reply:
done